src/django/api/templates/mail/claim_facility_submitted_body.html

```diff
@@ -2,59 +2,32 @@
 <html lang="en">
     <head>
         <title>
-            We received your request to claim an Open Supply Hub Facility
+            We received your request to claim a profile on Open Supply Hub
         </title>
     </head>
     <body>
         <p>
             Hi,
```

Comment:
There are some inconsistencies in the text/data we show the user in the email between the edited TXT file and this HTML file. Is this expected?
Edited TXT file is in this PR -> https://github.com/opensupplyhub/open-supply-hub/pull/268